include/ck_tile/ops/gemm/pipeline/gemm_pipeline_ag_bg_cr_comp_v4_default_policy.hpp

```diff
@@ -21,15 +21,28 @@ struct GemmPipelineAgBgCrCompV4DefaultPolicy
     CK_TILE_HOST_DEVICE static constexpr auto GetBlockGemm()
```

Comment:
should the pipeline methods be device-only?

Reply:
A lot of things are marked `CK_TILE_HOST_DEVICE` even if they are only used in device. Sometimes I choose to leave them there to avoid too much trivial changes in a PR for better experience of reviewing. BTW, #2475 tried to avoid protentional harmfulness of the misuse of `CK_TILE_HOST_DEVICE`.